From 3382f4de6ec9e9b83c5be39e7215788d71935ddb Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Thu, 19 Apr 2018 14:16:30 +0200
Subject: [PATCH 0592/1051] phy: add sata phy mode

Add SATA PHY mode to the phy_mode enum to allow configuring generic PHYs
to SATA by using set_mode callback.

Change-Id: I8f8b95a126c4e5a7e6933e9d8360c3692d2b978b
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 include/linux/phy/phy.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index a1480d76b800..cd8268a4c358 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -41,6 +41,7 @@ enum phy_mode {
 	PHY_MODE_UFS_HS_A,
 	PHY_MODE_UFS_HS_B,
 	PHY_MODE_PCIE,
+	PHY_MODE_SATA,
 };
 
 /**
-- 
2.17.1

